<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="DESKTOP-TQEA3MJ" name="GeneralAttributes" tests="7" failures="1" timestamp="20 Nov 2018 12:27:32 GMT" time="0.028" errors="1">
  <testcase name="m5" time="0.000" classname="GeneralAttributes">
    <skipped/>
  </testcase> <!-- m5 -->
  <testcase name="m9" time="0.002" classname="GeneralAttributes"/>
  <testcase name="m6" time="0.002" classname="GeneralAttributes"/>
  <testcase name="m8" time="0.001" classname="GeneralAttributes"/>
  <testcase name="m7" time="0.015" classname="GeneralAttributes">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method GeneralAttributes.m7() didn&#039;t finish within the time-out 10">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method GeneralAttributes.m7() didn't finish within the time-out 10
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m4" time="0.008" classname="GeneralAttributes">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at GeneralAttributes.m4(GeneralAttributes.java:21)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- m4 -->
  <testcase name="m3" classname="GeneralAttributes">
    <skipped/>
  </testcase> <!-- m3 -->
</testsuite> <!-- GeneralAttributes -->
